Algorithm.CSharp/DelistingFutureOptionDailyRegressionAlgorithm.cs

```diff
@@ -25,43 +26,50 @@ public class DelistingFutureOptionDailyRegressionAlgorithm : DelistingFutureOpti
     {
         protected override Resolution Resolution => Resolution.Daily;
 
+        protected override void PlaceOrder(Symbol symbol)
+        {
+            // We place a limit order because on daily resolution, data may come at a time when market is closed, so market orders are not allowed.
+            // Also, we use a very high limit price to ensure the order is filled right away with the next bar.
+            LimitOrder(symbol, 1, Securities[symbol].AskPrice * 2m);
+        }
+
         /// <summary>
         /// Data Points count of all timeslices of algorithm
         /// </summary>
-        public override long DataPoints => 15192;
+        public override long DataPoints => 13223;
 
         /// <summary>
         /// This is used by the regression test system to indicate what the expected statistics are from running the algorithm
         /// </summary>
         public override Dictionary<string, string> ExpectedStatistics => new Dictionary<string, string>
         {
-            {"Total Trades", "16"},
-            {"Average Win", "0.01%"},
-            {"Average Loss", "-0.02%"},
-            {"Compounding Annual Return", "-0.111%"},
-            {"Drawdown", "0.100%"},
-            {"Expectancy", "-0.678"},
-            {"Net Profit", "-0.111%"},
-            {"Sharpe Ratio", "-0.965"},
-            {"Probabilistic Sharpe Ratio", "0.000%"},
-            {"Loss Rate", "80%"},
-            {"Win Rate", "20%"},
-            {"Profit-Loss Ratio", "0.61"},
-            {"Alpha", "-0.001"},
-            {"Beta", "-0.001"},
-            {"Annual Standard Deviation", "0.001"},
+            {"Total Trades", "0"},
```

Comment:
Whops! trades went to 0?

Reply:
Solved, there was not enough data for the limit orders to be filled